inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.h

```diff
@@ -9,20 +9,21 @@
 #include <string>
 #include <memory>
 #include <vector>
+#include <common/memory_desc_wrapper.hpp>
 
 namespace MKLDNNPlugin {
 
 struct PortMap {
     // Data map rule
-    int from; /**< Index of external data from ins/outs fields of node */
-    int to;   /**< Index of internal data in iterator body */
+    int m_from; /**< Index of external data from ins/outs fields of node */
```

Comment:
What is "m_"? Please stick to general style.

Reply:
[comment](https://github.com/openvinotoolkit/openvino/pull/8879#discussion_r763102096)